modules/samples/firealarm/component/plugin/src/main/java/org/homeautomation/firealarm/plugin/constants/DeviceTypeConstants.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.homeautomation.firealarm.plugin.constants;
+
+public class DeviceTypeConstants {
+
+    public final static String DEVICE_TYPE = "firealarm";
+    public final static String DEVICE_PLUGIN_DEVICE_NAME = "DEVICE_NAME";
+    public final static String DEVICE_PLUGIN_DEVICE_ID = "firealarm_DEVICE_ID";
+    public final static String SENSOR_TEMPERATURE = "temperature";
+    public final static String SENSOR_HUMIDITY = "humidity";
+    public static final String DATA_SOURCE_NAME = "jdbc/firealarmDM_DB";
+
+    public final static String DEVICE_PLUGIN_PROPERTY_ACCESS_TOKEN = "accessToken";
+    public final static String DEVICE_PLUGIN_PROPERTY_REFRESH_TOKEN = "refreshToken";
```

Comment:
Most appropriate order of modifiers for defining a CONSTANT would be  -> 
public static final String DATA_SOURCE_NAME

Change others accordingly.